src/styles/main.scss

```diff
@@ -27,13 +27,16 @@ $day__names: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
     cursor: pointer;
     &:hover {
       transform: translateY(-20px);
-      background-color: pink;
+      background: #f4c4f3;
+      background: -webkit-linear-gradient(to right, #fc67fa, #f4c4f3);
+      background: linear-gradient(to right, #fc67fa, #f4c4f3);
     }
   }
+
   &--start-day {
     @each $name, $margin in $day__names {
       &--#{$name} :first-child {
-        margin-left: $margin * $size + $gap-day * $margin;
+        margin-left: $margin * $size + $gap-day * $margin + 1.6 * $margin;
```

Comment:
This formula works incorrectly, you have a shift in the first row.
![image](https://github.com/mate-academy/layout_calendar/assets/116377279/15406570-6044-4611-9897-3f04e2796430)